### PR TITLE
Jaymedatabase

### PR DESCRIPTION
**Pull request by jaymenelson56**

### CODE REVIEW

File: assets/campgrounds.jpg
Comment (ztrouy) [inline assets/campgrounds.jpg:1]: This file should not be in this branch. This is part of the assets directory branch.

File: assets/chamfort.jpg
Comment (ztrouy) [inline assets/chamfort.jpg:1]: This file should not be in this branch. This is part of the assets directory branch.

File: assets/ganderriver.jpg
Comment (ztrouy) [inline assets/ganderriver.jpg:1]: This file should not be in this branch. This is part of the assets directory branch.

File: assets/lodge.jpg
Comment (ztrouy) [inline assets/lodge.jpg:1]: This file should not be in this branch. This is part of the assets directory branch.

File: assets/lostwolf.jpg
Comment (ztrouy) [inline assets/lostwolf.jpg:1]: This file should not be in this branch. This is part of the assets directory branch.

File: assets/pinebluff.jpg
Comment (ztrouy) [inline assets/pinebluff.jpg:1]: This file should not be in this branch. This is part of the assets directory branch.

File: scripts/database.js
Comment (ztrouy) [inline scripts/database.js:1]: This looks good to me

**Review by ztrouy — CHANGES_REQUESTED** [review]
This includes the assets that are up for approval in another branch. Please remove any changes this branch makes to the assets directory.

The exported functions in database.js look fine to me

File: assets/campgrounds.jpg
Comment (jaymenelson56) [reply assets/campgrounds.jpg:1]: Removed it.

**Review by ztrouy — APPROVED** [review]
This now looks good to me, the errant files were removed

**Review by Michael-Deitz — APPROVED** [review]
after the changes made this now looks good to me

**Review by AngRod96 — APPROVED** [review]

**Pull request merged**